### PR TITLE
Add Material_SpecularGlossiness test set

### PR DESCRIPTION
**Pull request by stevk**
Adds 16 new models testing the SpecGloss extension.

### CODE REVIEW

File: Source/AssetGenerator.csproj [hunk 1]
```diff
@@ -96,13 +96,13 @@
     <EmbeddedResource Include="Tests\lambert2_baseColor.png" />
     <EmbeddedResource Include="Tests\lambert2_emissive.png" />
     <EmbeddedResource Include="Tests\lambert2_normal.png" />
-    <EmbeddedResource Include="Tests\lambert2_occlusionRoughnessMetallic.png" />
+    <EmbeddedResource Include="Tests\lambert2_MetallicRoughness.png" />
```
Comment (bghgary) [inline Source/AssetGenerator.csproj]: should be lower camel case to match other textures (i.e. MetallicRoughness -> metallicRoughness)

**Comment by kcoley** [comment]
I think you will need to set the specularFactor to [0, 0, 0] when you generate test models that are not using specular.  Otherwise, the specularFactor defaults to [1,1,1] and overrides the diffuse texture

File: Source/Test.cs [hunk 2]
```diff
@@ -16,7 +16,9 @@ internal class Test
         public const string texture_Normal = "lambert2_normal.png";
         public const string texture_Emissive = "lambert2_emissive.png";
         public const string texture_BaseColor = "lambert2_baseColor.png";
-        public const string texture_OcclusionRoughnessMetallic = "lambert2_occlusionRoughnessMetallic.png";
+        public const string texture_MetallicRoughness = "lambert2_metallicRoughness.png";
```
Comment (bghgary) [inline Source/Test.cs]: Maybe we should separate out occlusion texture into its own texture.

**Comment by stevk** [comment]
Is this what we had in mind for occlusion?

image

**Comment by stevk** [comment]
image

**Comment by stevk** [comment]
- [ ] Only test float-vec3 vertex color in metallicRough/specgloss
- [ ] Test 4 vertex color in alpha
- [ ] Large pinapple

**Comment by stevk** [comment]
MetallicRoughness
image

Spec Gloss
image

Alpha
image

**Pull request merged**